tools/scripts/utils.mjs

```diff
@@ -566,6 +611,39 @@ export const log = Object.assign(
   }
 );
 
+export const testThis = async (test) => {
+  // calculate the time it takes to run the test
+  const start = Date.now();
+
+  const { status, message } = await test.fn();
```

Comment:
should we add a `catch` to prevent unresolved rejections?

Reply:
it's already in a try/catch block though?